### PR TITLE
Diagram assignment

### PR DESCRIPTION
**Pull request by yossimyhill**
Hi Laura.
Here are the diagrams you asked for.
Hope they are good:)

### CODE REVIEW

**Comment by lsnovich** [comment]
Hi Yossi
Please make all lines in the flowchart solid lines. 
Thanks!

File: Morning_Schedule_Flowchart.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# Morning Schedule Flowchart
+```mermaid
+flowchart TD;
+
+A((START)) --> B[Wake up];
+B --> C[Wash hands, get dressed, say morning blessings];
+C --> D{What time is it?};
+D -.Before 06:00.-> E[Go to Mikva];
```
Comment (lsnovich) [inline Morning_Schedule_Flowchart.md:8]:
```suggestion
D --Before 06:00--> E[Go to Mikva];
```

File: Morning_Schedule_Flowchart.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+D -.After 06:00.-> F{Is it before 06:30?};
```
Comment (lsnovich) [inline Morning_Schedule_Flowchart.md:9]:
```suggestion
D --After 06:00--> F{Is it before 06:30?};
```

File: Morning_Schedule_Flowchart.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+E-->G[Go to Shul];
+F-.Yes.->G;
```
Comment (lsnovich) [inline Morning_Schedule_Flowchart.md:11]:
```suggestion
F--Yes-->G;
```

File: Morning_Schedule_Flowchart.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+F-.No.->H[Help the kids get ready for school];
```
Comment (lsnovich) [inline Morning_Schedule_Flowchart.md:12]:
```suggestion
F--No-->H[Help the kids get ready for school];
```

File: Morning_Schedule_Flowchart.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+H-->I[Bring the kids to school];
+I-->G;
+G-->J[Come home, eat breakfast, get snacks and lunch ready];
+J-->K{Are the kids in school yet?};
+K -.Yes.-> M{Is there time to clean the house before leaving?};
```
Comment (lsnovich) [inline Morning_Schedule_Flowchart.md:17]:
```suggestion
K --Yes--> M{Is there time to clean the house before leaving?};
```

File: Morning_Schedule_Flowchart.md [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+K -.No.-> L[Finish getting them ready, bring them to school];
```
Comment (lsnovich) [inline Morning_Schedule_Flowchart.md:18]:
```suggestion
K --No--> L[Finish getting them ready, bring them to school];
```

File: Morning_Schedule_Flowchart.md [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+M-.Yes.->N[Clean the house];
```
Comment (lsnovich) [inline Morning_Schedule_Flowchart.md:19]:
```suggestion
M--Yes-->N[Clean the house];
```

File: Morning_Schedule_Flowchart.md [hunk 8]
```diff
@@ -0,0 +1,32 @@
...
+M-.No.->O[Walk towards the nearest number 12 bus stop];
```
Comment (lsnovich) [inline Morning_Schedule_Flowchart.md:20]:
```suggestion
M--No-->O[Walk towards the nearest number 12 bus stop];
```

File: Morning_Schedule_Flowchart.md [hunk 9]
```diff
@@ -0,0 +1,32 @@
...
+L-->O;
+N-->O;
+O-->P[Wait on the bus until the Hagalil St. stop];
+P-->Q{Is there a number 26 bus coming to the stop within 15 minutes?};
+Q-.Yes.->R[Get off the 12 at Hagalil, wait for the 26, go onto the 26];
```
Comment (lsnovich) [inline Morning_Schedule_Flowchart.md:25]:
```suggestion
Q--Yes-->R[Get off the 12 at Hagalil, wait for the 26, go onto the 26];
```

File: Morning_Schedule_Flowchart.md [hunk 10]
```diff
@@ -0,0 +1,32 @@
...
+R-->S[Get off at Eli Cohen St.];
+Q-.No.->T[Stay on the 12 until the Highway 40 stop, get off at that stop];
```
Comment (lsnovich) [inline Morning_Schedule_Flowchart.md:27]:
```suggestion
Q--No-->T[Stay on the 12 until the Highway 40 stop, get off at that stop];
```

**Review by lsnovich — CHANGES_REQUESTED** [review]
please accept or reject changes

**Review by lsnovich — COMMENTED** [review]
The apartment rental and morning schedule files are in the wrong location.
Please move them to 
student-work/2023/Yosef-Myhill/diagrams

**Comment by lsnovich** [comment]
Good start! If my comments or changes are not clear let me know.

**Review by Laura-Novich-OBW — APPROVED** [review]

**Pull request merged**